config/checker_severity_map.json

```diff
@@ -55,6 +59,7 @@
   "bugprone-misplaced-widening-cast":                           "HIGH",
   "bugprone-move-forwarding-reference":                         "MEDIUM",
   "bugprone-multiple-statement-macro":                          "MEDIUM",
+  "bugprone-narrowing-conversions":                             "MEDIUM", 
```

Comment:
will be renamed in clang9, I did not found `bugprone-narrowing-conversions` only `cppcoreguidelines-narrowing-conversions`

Reply:
put to cppcoreguidelines only, and only to extreme level.